js/pages/samples.js

```diff
@@ -325,23 +325,58 @@ const sanford = {
+    whenToDonate: 'The Connect team will send a MyChart message when it is time to donate your samples. If you do not have a MyChart account, we will send you an email. Be sure to check your spam or junk folder. After you receive the MyChart message or email, it is important to donate your samples as soon as you can.<br><br><span style="font-weight:900; text-decoration:underline">Note:</span> If you have recently had a blood transfusion or donated blood, please wait at least <span class="site-info-bold">eight weeks</span> from your donation or transfusion before donating your samples for Connect. If you have recently donated plasma, please wait at least <span class="site-info-bold">two days</span> from your plasma donation before donating samples for Connect. If you have an upcoming colonoscopy, please be sure that you <span class="site-info-bold">do not</span> donate samples for Connect on the <span class="site-info-bold">same day</span> as your colonoscopy.',
+    howToDonate: 'Connect participants at Sanford Health have two options for donating samples. You can choose the most convenient option for you.<br><br> Option 1: Sanford Health Lab Location <br><br> Once you receive our message, you may walk in to donate samples at any participating <span class="site-info-bold">Sanford Health Lab Location</span> during normal hours of operation. <span class="site-info-bold">You do not need to schedule an appointment</span>. You are welcome to donate your samples for Connect at the same time as any prescheduled medical appointments to save you a trip to the lab. <br><br> <span style="font-weight:900; text-decoration:underline">Note:</span>  If you live in the Bismarck region, the Sanford lab team prefers you pre-schedule an appointment. You can do this within your MyChart or by calling the lab directly.<br><br> Option 2: Connect Research Lab <br><br>  <span class="site-info-bold">Make an appointment</span> to come into one of our <span class="site-info-bold">Connect Research Labs</span> to donate your samples. <br><br> The table below includes more information about these options. <br><br>' +
+        '<table style="border: 1px solid">' +
+        '<tr style="border: 1px solid">' +
```

Comment:
Should these separate lines merge to one string like other string values?

Reply:
I put it on multiple lines so we can easily see and update the code.